packages/config-schema/src/schema/netlify-toml.schema.json

```diff
@@ -0,0 +1,722 @@
+          }
+        },
+        "additionalProperties": false,
+        "required": ["path", "function"]
+      },
+      "x-taplo": {
+        "links": {
+          "key": "https://docs.netlify.com/edge-functions/declarations/"
+        }
+      }
+    },
+    "dev": {
+      "title": "Netlify Dev",
+      "description": "Netlify Dev enables a local development environment without any additional setup and this includes using the functions directory setting to scaffold and serve your functions locally. You can use the `[dev]` section to configure this local development environment. \n\nNote that `[dev]` doesn't run in the Bash shell, so you won't be able to use Bash-compatible syntax with the command.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "command": {
+          "title": "Dev command",
+          "description": "Command that starts your development server or runs a command such as a compiler watch in the background. If no `targetPort` is specified, it runs the command in the background in addition to the static file server.",
+          "type": "string"
+        },
+        "port": {
+          "title": "Port",
+          "description": "Port that Netlify Dev is accessible from in the browser.",
+          "type": "number",
+          "default": 8888
+        },
+        "targetPort": {
+          "title": "Target port",
+          "description": "Port for your application server, framework, or site generator. If provided, the CLI will wait until the provided `targetPort` is reachable and then proxy requests to it. If you specify values for both `command` and `targetPort`, `framework` must be `#custom`.",
+          "type": "number"
+        },
+        "functionsPort": {
+          "title": "Functions port",
+          "description": "Port where Netlify Dev serves functions.",
+          "type": "number"
+        },
+        "publish": {
+          "title": "Publish path",
+          "description": "Path to your static content folder.",
+          "type": "string"
+        },
+        "jwtRolePath": {
+          "title": "JWT Role Path",
+          "description": "Object path that points to role values for JWT-based redirects.",
+          "type": "string"
+        },
+        "jwtSecret": {
+          "title": "JWT Secret",
+          "description": "Secret used to verify tokens for JWT-based redirects.",
+          "type": "string"
+        },
+        "autoLaunch": {
+          "title": "Publish path",
+          "description": "Boolean value that determines whether Netlify Dev launches the local server address in your browser.",
+          "type": "boolean"
+        },
+        "framework": {
+          "title": "Publish path",
+          "description": "Setting to use if a project is detected incorrectly, flagged by multiple detectors, or requires a `command` and `targetPort`.",
+          "type": "string",
+          "enum": ["#auto", "#static", "#custom"],
```

Comment:
should have a regular string here as well like `next`

Reply:
Interesting, it seems the docs need an update then too maybe.

Reply:
I removed the enum and made it a simple string for now. LAter one we could make it an enum again, but generate it with all the framework names :)